unity/Assets/StreamingAssets/content/MoM/soa/items.ini

```diff
@@ -101,17 +101,17 @@ traits=relic unique
 [ItemSpellAzureFlame]
 name={ffg:SPELL_AZURE_FLAME}
 image={import}/img/Spell_AzureFlame.dds
```

Comment:
Please remove the .dds file extension from all entries

Reply:
This change is targetting master, which still has the .dds, and it isn't part of this change.  Merging into dev solves this.  Same for other pull requests.

Reply:
I am very slowly learning something of the ins and outs of Valkyrie.
The image setting contains what looks like the correct name as it matches the file name in the import folder.
In the import folder, the images for the spells seem to me to be dds files.
So why does dev need to change it to something else on the merge?

Reply:
To explain @redwolf2 's comment further:

redwolf2 has been working on adding Android support to Valkyrie.  When importing from Android the image assets are created as .pvr files and not .dds files.  To work around the ini files being wrong, the fix was to remove the .dds from all of these files and let Valkyrie find any matching images.

This has been done on the development branch.  You change is separate to this, but because this pull request is targeting the master branch (1.6.1 release) it shows the (now outdated) .dds extensions.  I assume redwolf2 thought you were adding them back in due to having made an earlier copy.

I have added your change to the development branch and it will be in the next Valkyrie release.  Thank you.

Reply:
Jepp sry, got confused